osu.Game/Skinning/FontAdjustableSkinComponent.cs

```diff
@@ -20,11 +21,16 @@ public abstract partial class FontAdjustableSkinComponent : Container, ISerialis
         [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.Font), nameof(SkinnableComponentStrings.FontDescription))]
         public Bindable<Typeface> Font { get; } = new Bindable<Typeface>(Typeface.Torus);
 
+        [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.FontColour), nameof(SkinnableComponentStrings.FontColourDescription))]
+        public BindableColour4 FontColour { get; } = new BindableColour4(Colour4.White);
```

Comment:
Please rename this (and all associated members) to `TextColour`. Fonts don't have colours.